chunky/src/java/se/llbit/chunky/world/EmptyWorld.java

```diff
@@ -31,8 +38,23 @@ private EmptyWorld() {
     this.currentDimension = EmptyDimension.INSTANCE;
   }
 
+  @Override
+  public Set<Integer> listDimensions() {
```

Comment:
Why would dimensions be integers? Is that a sane assumption or just some Java detail?